### PR TITLE
Updates needed for running test on windows inbox

### PR DESCRIPTION
**Pull request by bingbing8**
1. add quotes to the module path
2. Invoke-OpenSSHE2ETest return test results which can be consumed by other logger, like wtt.

### CODE REVIEW

File: contrib/win32/openssh/OpenSSHTestHelper.psm1 [hunk 1]
```diff
@@ -506,13 +506,18 @@ function Get-UnitTestDirectory
     Run OpenSSH pester tests.
 #>
 function Invoke-OpenSSHE2ETest
-{     
+{
+    [CmdletBinding()]
+    param
+    (
+        [ValidateSet('CI', 'Scenario')]
```
Comment (bagajjal) [inline contrib/win32/openssh/OpenSSHTestHelper.psm1:513]: when do we use 'CI' and 'scenario'?

File: regress/pesterTests/KeyUtils.Tests.ps1 [hunk 2]
```diff
@@ -330,7 +352,7 @@ Describe "E2E scenarios for ssh key management" -Tags "CI" {
         }
     }
 		
-    Context "$tC - ssh-keyscan test cases" {
+    <#Context "$tC - ssh-keyscan test cases" {
```
Comment (bagajjal) [inline regress/pesterTests/KeyUtils.Tests.ps1]: Plz disable the ssh-keyscan only for windowsInbox...

File: contrib/win32/openssh/OpenSSHTestHelper.psm1 (hunk 1 above)
Comment (bingbing8) [reply contrib/win32/openssh/OpenSSHTestHelper.psm1:513]: It is the priority of the tests. CI is continuous integration test which is basic tests required to run for each build. While Scenario is the scenario tests which may be run less frequent.

File: regress/pesterTests/KeyUtils.Tests.ps1 (hunk 2 above)
Comment (bingbing8) [reply regress/pesterTests/KeyUtils.Tests.ps1]: fixed in latest version

**Pull request merged**